app/controllers/cinema_halls_controller.rb

```diff
@@ -1,15 +1,53 @@
 # frozen_string_literal: true
 
 class CinemaHallsController < ApplicationController
+  before_action :set_cinema_halls, only: %i[show update destroy]
+  # GET /cinema_halls
   def index
-    cinema_halls = CinemaHall.all
-    cinema_halls = CinemaHall.where('name like ?', "%#{params[:name]}%") if params[:name]
+    Sentry.capture_message('test message')
```

Comment:
I guee that it's provided just to test communication with sentry, but it's still relevant?

Reply:
No I use it to check if it is working, will delete it.